Rakefile

```diff
@@ -19,6 +19,12 @@ namespace :test do
     t.libs << 'test'
     t.verbose = true
   end
+
```

Comment:
How about adding this task's pattern to the `:dotcom` task above? Or is this temporary until you fix these tests?